store/tikv/2pc.go

```diff
@@ -480,29 +515,22 @@ func (c *twoPhaseCommitter) doActionOnBatches(bo *Backoffer, action twoPhaseComm
 	return errors.Trace(err)
 }
 
-func (c *twoPhaseCommitter) keyValueSize(key []byte) int {
-	size := len(key)
-	if mutation := c.mutations[string(key)]; mutation != nil {
-		size += len(mutation.Value)
-	}
-	return size
+func (c *twoPhaseCommitter) keyValueSize(key, value []byte) int {
+	return len(key) + len(value)
 }
 
-func (c *twoPhaseCommitter) keySize(key []byte) int {
+func (c *twoPhaseCommitter) keySize(key, value []byte) int {
```

Comment:
Why change it?

Reply:
The grouping function now can handle both keys or mutations. So the size function takes both key and values. This change is to conform that signature change.